examples/async-serial-luba.py

```diff
@@ -77,9 +123,13 @@ async def listen_luba():
                 except DALISequenceError:
                     pass
 
+    # Listen and print out any intercepted DALI commands
     print("\nListening for DALI commands on the bus...\n")
     while True:
         cmd = await driver.wait_dali_rx()
+        if isinstance(cmd, AmbiguousInstanceType):
+            if new_cmd := cmd.retry_decode(dev_inst_map):
+                cmd = new_cmd
```

Comment:
Can the `DeviceInstanceTypeMapper` be passed to the driver instance so it can use it when decoding frames received on the bus?

Reply:
Oh yeah good idea, I'll have a go at implementing that (at least in the LUBA serial driver I wrote, and also the HID driver - those are the only two I can test with)

Reply:
OK so I've been taking a look at this - I've got the decoding added into my LUBA driver, but I tried adding it to the Tridonic HID driver only to find that it doesn't seem to support 24 bit commands. Does that sound right to you @sde1000? I suppose the interface predates the DALI-2 specification, so it makes sense it doesn't support it.